analyzer/src/main/resources/scripts/init.gradle

```diff
@@ -448,6 +501,32 @@ class AbstractDependencyTreePlugin<T> implements Plugin<T> {
             return identifierFromDisplayName(dependencyResult.requested.displayName)
         }
 
+        /**
+         * Find the root project of an included build. The latter is identified by the given Identifier of an included
+         * project.
+         * @param project the original project for which the dependency tree is built. Used only for logging purposes.
+         * @param identifier the identifier of an included project.
+         * @return the root project of the build of the included project.
```

Comment:
Nit: Start sentences with an uppercase letter.

Reply:
All the other parameters documentation in this file don't begin with an uppercase letter. I did it like that for consistency.

Reply:
Yes, but the other parameter docs do not end with dots.

Reply:
Understood, I removed the dots.